lms/templates/instructor/instructor_dashboard_2/certificate-white-list-editor.underscore

```diff
@@ -2,11 +2,14 @@
 <p class="under-heading"> <%- gettext("Enter the username or email address of each learner that you want to add as an exception.") %></p>
 <div class='certificate-exception-inputs'>
     <div class="">
-    <input class='student-username-or-email' id="certificate-exception" type="text" placeholder="Student email or username" aria-describedby='student-user-name-or-email-tip'>
-    <textarea class='notes-field' id="notes" rows="10" placeholder="Free text notes" aria-describedby='notes-field-tip'></textarea>
+        <label for="certificate-exception" class="sr"><%- gettext("Student email or username") %></label>
```

Comment:
So in this file, you have "sr" labels with equivalent placeholder text. In certificate-invalidation.underscore, you went with visible labels and no placeholder text. Should the same approach be used in both?